src/Esfa.Vacancy.Register.UnitTests/SearchApprenticeship/Application/GivenVacancySearchParametersMapper/AndFrameworkCodes.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancies;
+using FluentAssertions;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Esfa.Vacancy.Register.UnitTests.SearchApprenticeship.Application.GivenVacancySearchParametersMapper
+{
+    [TestFixture]
+    public class AndFrameworkCodes
```

Comment:
I suggest the folder would be 
Given**A**VacancySearchParametersMapper

and the test files would be
WhenMappingFrameworkCodes and WhenMappingStandardCodes